tests/unit/LedgerConsentContinuationOut.spec.ts

```diff
@@ -20,13 +20,15 @@ describe('Display ConsentContinuationOut component', () => {
         }
```

Comment:
There's a console warning on the mount commands in this test suite, "[Vuetify] The v-expansion-panel component must be used inside a v-expansion-panels".

I tried a few things to fix this test issue (is not a component issue), using `parentComponent`, but it didn't quite work and I've spend enough time on it for now.

So just ignore these test warnings.

Reply:
Alright, noted Sev!